app/src/main/java/com/standuptracker/MainActivity.kt

```diff
@@ -1,18 +1,18 @@
 package com.standuptracker
 
 import android.os.Bundle
-import com.google.android.material.floatingactionbutton.FloatingActionButton
-import com.google.android.material.snackbar.Snackbar
+import android.view.Menu
```

Comment:
According to android best practices, imports should be alphabetized

Reply:
Very cool.

Some keyboard shortcuts to go along: 

Control-Alt-O: Organize imports
Control-Alt-L: Reformat code
Control-Alt-I: Fix indents.